src/data/CassandraBackend.hpp

```diff
@@ -950,7 +950,7 @@ class BasicCassandraBackend : public BackendInterface {
         std::vector<Statement> statements;
         statements.reserve(data.size());
         for (auto [mptId, holder] : data)
-            statements.push_back(schema_->insertMPTHolder.bind(std::move(mptId), std::move(holder)));
+            statements.push_back(schema_->insertMPTHolder.bind(mptId, holder));
```

Comment:
I don't think it is a related change.

Reply:
It's not. it's a drive-by fix as stated in the description

Reply:
Fine, i'll move it to the other branch i have which is `clang-tidy` related